README.md

```diff
@@ -14,15 +14,15 @@
 
 ## Description
 
-The Puppet wordpress_app is a module that demonstrates an example application model. The module contains application components you can use to set up a WordPress database, a PHP application server, and an HAProxy load balancer. With these components, you can build two WordPress applications: a simple LAMP stack or a stack that uses complex load-balancing.
+The Puppet wordpress_app is a module that demonstrates an example application model. The module contains application components you can use to set up a WordPress database, a PHP application server, and an HAProxy load balancer. With these components, you can build two WordPress applications: a simple LAMP stack or a complex stack that uses load-balancing.
 
 ## Setup Requirements
```

Comment:
I'm commiting a Puppetfile to this repo. Can you add something like:

You can install the module and it's dependencies with the included `Puppetfile`.